gems/quilt_rails/lib/generators/sewing_kit/templates/sewing-kit.config.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-env node */
+
+import * as path from 'path';
+
+import {Plugins, Env} from '@shopify/sewing-kit';
+import {ReactServerPlugin} from '@shopify/react-server-webpack-plugin';
+
+const root = path.resolve(__dirname, '../');
+const tests = path.resolve(root, 'app', 'ui', 'test');
+const sourceRoot = path.resolve(root, 'app', 'ui');
+
+module.exports = function sewingKitConfig(plugins: Plugins, env: Env) {
```

Comment:
we can basically make this file only the project name with our latest version of SK

Reply:
Project name does absolutely nothing, so is this needed at all?

Reply:
probably not unless we want to show people they *can* configure stuff

Reply:
We could have it empty with a bunch of commented out fields for example, similar to the `tsconfig` you get from `tsc init`

Reply:
Went with a minimal config, we can do small iterations on this.